k2/torch/csrc/ragged_any.h

```diff
@@ -20,15 +21,16 @@
  * limitations under the License.
  */
 
-#ifndef K2_PYTHON_CSRC_TORCH_V2_RAGGED_ANY_H_
-#define K2_PYTHON_CSRC_TORCH_V2_RAGGED_ANY_H_
+#ifndef K2_TORCH_CSRC_RAGGED_ANY_H_
+#define K2_TORCH_CSRC_RAGGED_ANY_H_
+
 #include <string>
 #include <tuple>
 #include <utility>
 #include <vector>
 
 #include "k2/csrc/ragged.h"
-#include "k2/python/csrc/torch.h"
+#include "torch/torch.h"
```

Comment:
Please use `torch/script.h`, not `torch/torch.h`, if you want to remove Python dependencies.